packages/javascript-sdk/src/token-storage/local-storage.ts

```diff
@@ -18,19 +18,20 @@ abstract class LocalStorageWrapper {
   /**
    * Retrieve tokens.
    */
-  public static async get(clientId: string): Promise<Tokens> {
-    const tokenString = localStorage.getItem(`${DB_NAME}-${clientId}`);
+  public static async get(clientId: string): Promise<Tokens | void> {
+    const tokenString = localStorage.getItem(`${PREFIX}-${clientId}`);
+
+    // If there is no stored token, or the token is not an object, return null
+    if (!tokenString) {
+      // This is a normal state, so resolve with undefined
+      return;
```

Comment:
Decided to keep the function `async`, but just remove the `Promises` to keep it cleaner.